public/js/gamepiece.js

```diff
@@ -8,7 +8,16 @@
     }
     
```

Comment:
The player variables that were added to gamepiece on master don't seem to be here?